charts/karpenter/templates/100-config-logging.yaml

```diff
@@ -31,6 +31,3 @@ data:
         "timeEncoder": "iso8601"
       }
     }
-  # Log level overrides
-  loglevel.controller: info # debug
```

Comment:
I think it would be good to just comment these two lines out (completely) - that way people can see how to change the log level if they wish.